CONTRIBUTING.md

```diff
@@ -1,22 +1,79 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Contributing to the Ferdium Website
 
-## Getting Started
+:tada: First off, thanks for taking the time and your effort to make Ferdium better! :tada:
+
+## Table of contents
+
+<!-- TOC depthFrom:2 depthTo:2 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Table of contents](#table-of-contents)
+- [Code of Conduct](#code-of-conduct)
+- [How do I contribute?](#how-do-i-contribute)
+	- [Setup your development environment](#setup-your-development-environment)
+		- [Node.js, pnpm](#nodejs-pnpm)
+		- [Git](#git)
+	- [Clone the repository](#clone-the-repository)
+	- [Install dependencies](#install-dependencies)
+	- [Add your modifications](#add-your-modifications)
+	- [Build the website locally](#build-the-website-locally)
+- [Learn More](#learn-more)
+
+<!-- /TOC -->
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Ferdium Code of Conduct](https://raw.githubusercontent.com/ferdium/ferdium-app/develop/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
+Please report unacceptable behavior to [hello@ferdium.org](mailto:hello@ferdium.org).
+
+## How do I contribute?
+
+### Setup your development environment
+
+#### Node.js, pnpm
+
+You will need a working installation of Node.js, with the `pnpm` package manager.
+It is advised to use the version recommended in the [`ferdium-app` repository](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md#nodejs-npm-pnpm).
```

Comment:
Same here - the 2 repos might evolve independently of each other (though, as maintainers we would try to keep them in sync as to node/npm/pnpm versions). Is that ok or do we want to keep the documentation also independent?

Reply:
I think it's okey to link the documentation and code of conduct of the main repo: it will be easier to keep it in sync if there is only one copy.
For the website, it's not really important to pin the version, as it is for the other repositories. It's why it's not enforced in `package.json#engines`. As long as we stay on a recent version of node, NextJS should work.